src/Microsoft.Health.Fhir.Core/Features/Security/Role.cs

```diff
@@ -3,41 +3,22 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
-using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations;
-using System.Globalization;
+using EnsureThat;
 
 namespace Microsoft.Health.Fhir.Core.Features.Security
 {
-    public class Role : IValidatableObject
+    public class Role
```

Comment:
Wondering if we should add scope to `Role` even though it would always be `{ "/" }`. This to make it clearer later when we get to using it.